autoscale-scaler-dockerswarm/pom.xml

```diff
@@ -51,17 +51,6 @@
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
         </dependency>
-        <dependency>
-            <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-simple</artifactId>
-            <version>1.7.12</version>
-            <scope>test</scope>
```

Comment:
The removal of `slf4j-simple` as a test-scope dependency surprises me.  Do the tests still log following this removal?

Reply:
There is log after removing this dependency. 
Looks DockerSwarmServiceScalerIT tests are not executed and there is no test results for this class in log. 
http://sou-jenkins2.hpeswlab.net/job/Autoscaler/view/Developer/job/Autoscaler~autoscaler~SCMOD-5060~CI/50/consoleFull

Reply:
OK.  I don't know why the tests are set to not execute by default but I still want this dependency put back so that they will log if they are executed.

Reply:
The slf4j-simple added back for logging in future for tests.